src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/OnlineHearingService.java

```diff
@@ -1,35 +1,62 @@
 package uk.gov.hmcts.reform.sscscorbackend.service;
```

Comment:
Same comments as EvidenceUploadService maybe is we split this into two as PDF and OnlineHearingService we could break the circular dependency.